pages/list.html

```diff
@@ -4,10 +4,19 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../public/css/list.css">
     <title>List</title>
 </head>
 <body>
-    <a href="../index.html">Home</a>
+    
+    <div class="list-section">
+        <div class="list-section_navigation">
+        <a href="../index.html" class="Home">Home</a>
```

Comment:
Change class name from Home to home, no big letter at begging of class

Reply:
Ok